MDC2020/gen_Mix.sh

```diff
@@ -80,7 +81,7 @@ echo physics.filters.MuStopPileupMixer.mu2e.MaxEventsToSkip: ${nskip_MuStopPileu
 # setup database access for SimEfficiencies
 #
 echo 'services.ProditionsService.simbookkeeper.useDb: true' >> template.fcl
-echo services.DbService.purpose: $2$4 >> template.fcl
+echo services.DbService.purpose: $2$3 >> template.fcl
```

Comment:
Also here is the version of the primary or the version of the beam samples?

Reply:
You are right, the primary defines the output version.  I reverted these changes